seatunnel-server/seatunnel-app/src/main/resources/application.yml

```diff
@@ -52,7 +52,13 @@ jwt:
   secretKey:
   algorithm: HS256
 
-
+seatunnel-web:
```

Comment:
In an earlier PR, we placed the Seatunnel-web specific configuration under the spring config section. 
You have now put seatunnel-web specific configuration under the seatunnel-web config section.
 This seems to be a better approach. Later, we can also move the configurations from the earlier PR under the seatunnel-web config section.